nbgrader/converters/base.py

```diff
@@ -43,9 +43,21 @@ class BaseConverter(LoggingConfigurable):
         )
     ).tag(config=True)
 
+    groupshared = Bool(
```

Comment:
I think this should probably be an attribute of CourseDirectory, and then accessed via `self.coursedir.groupshared`, so that the config value only needs to be set once.

Reply:
done, thanks.